src/pr-info.ts

```diff
@@ -385,26 +385,30 @@ const configSuspicious = <ConfigSuspicious>(async (path, getContents) => {
     if (text === undefined) return `couldn't fetch contents`;
     const tester = configSuspicious[basename];
     let suspect: string | undefined;
-    if (tester.length === 1) {
+    if (tester.length <= 1) {
         suspect = tester(text);
     } else {
         const oldText = await getContents("master");
         suspect = tester(text, oldText);
     }
     return suspect;
 });
-configSuspicious["OTHER_FILES.txt"] = contents =>
-    // not empty
-    (contents.length === 0) ? "empty"
-    : undefined;
+configSuspicious["OTHER_FILES.txt"] = makeJsonCheckerFromCore(
+    [],
+    [],
+    text => text.split(/\r?\n/),
```

Comment:
:+1: Yeah, this is also adding more stuff to the config checking thing, but I really like this one since it's pulling in `OTHER_FILES.txt` in a nice way too.

*However* the `Json` in the name is now bogus.  How about re organizing it a bit:
* Call it `makeChecker` (the `FromCore` is also kind of stale now, IMP)
* Make it take the parser as a last argument, and make it default to `JSON.parse`

?

Reply:
Done :heavy_check_mark: